base/strings/string.jl

```diff
@@ -425,6 +425,17 @@ function reverse(s::String)
     String(buf)
 end
 
+"""
+    repeat(s::String, r::Integer)
+
+Repeat a string `r` times.
```

Comment:
But I'm not sure why anyone would ever call `repeat` for a string rather than using `^`.